packages/ses/src/error/console.js

```diff
@@ -343,21 +356,26 @@ export { makeCausalConsole };
 /** @type {FilterConsole} */
 const filterConsole = (baseConsole, filter, _topic = undefined) => {
   // TODO do something with optional topic string
-  const whilelist = consoleWhitelist.filter(([name, _]) => name in baseConsole);
-  const methods = whilelist.map(([name, severity]) => {
+  const whitelist = arrayFilter(
+    consoleWhitelist,
+    ([name, _]) => name in baseConsole,
+  );
+  const methods = arrayMap(whitelist, ([name, severity]) => {
     /**
      * @param {...any} args
      */
     const method = (...args) => {
+      // eslint-disable-next-line @endo/no-polymorphic-call
       if (severity === undefined || filter.canLog(severity)) {
         // @ts-ignore
+        // eslint-disable-next-line @endo/no-polymorphic-call
         baseConsole[name](...args);
       }
     };
     return [name, freeze(method)];
   });
   const filteringConsole = fromEntries(methods);
-  return freeze(filteringConsole);
+  return /** @type {VirtualConsole} */ (freeze(filteringConsole));
```

Comment:
This is an unrelated change that was necessary to appease the IDE type-checker, but not our TSC lint CI job.